src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar_start-day--mon calendar_month-length--31">
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
+      <div class="calendar__day calendar__day--4"></div>
+      <div class="calendar__day calendar__day--5"></div>
+      <div class="calendar__day calendar__day--6"></div>
+      <div class="calendar__day calendar__day--7"></div>
+      <div class="calendar__day calendar__day--8"></div>
+      <div class="calendar__day calendar__day--9"></div>
+      <div class="calendar__day calendar__day--10"></div>
+      <div class="calendar__day calendar__day--11"></div>
+      <div class="calendar__day calendar__day--12"></div>
+      <div class="calendar__day calendar__day--13"></div>
+      <div class="calendar__day calendar__day--14"></div>
+      <div class="calendar__day calendar__day--15"></div>
+      <div class="calendar__day calendar__day--16"></div>
+      <div class="calendar__day calendar__day--17"></div>
+      <div class="calendar__day calendar__day--18"></div>
+      <div class="calendar__day calendar__day--19"></div>
+      <div class="calendar__day calendar__day--20"></div>
+      <div class="calendar__day calendar__day--21"></div>
+      <div class="calendar__day calendar__day--22"></div>
+      <div class="calendar__day calendar__day--23"></div>
+      <div class="calendar__day calendar__day--24"></div>
+      <div class="calendar__day calendar__day--25"></div>
+      <div class="calendar__day calendar__day--26"></div>
+      <div class="calendar__day calendar__day--27"></div>
+      <div class="calendar__day calendar__day--28"></div>
+      <div class="calendar__day calendar__day--29"></div>
+      <div class="calendar__day calendar__day--30"></div>
+      <div class="calendar__day calendar__day--31"></div>
```

Comment:
Do not hard code modifier in html
use [nth-child](https://css-tricks.com/how-nth-child-works/))